README.markdown

```diff
@@ -2,7 +2,16 @@
 Github-Viewer.slack
 ======================
 
-LICENSE
```

Comment:
I think that it is better that there is "LICENSE" statement in readme.
Please write into `MIT LICENSE` statement that is linked to the LICENSE file.